server/game/cards/01-DTR/CheatinVarmint.js

```diff
@@ -0,0 +1,31 @@
+const ActionCard = require('../../actioncard.js');
+
+class CheatinVarmint extends ActionCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Resolution: Lower hand by 2 ranks',
+            playType: ['resolution'],
+            choosePlayer: true,
+            cost: ability.costs.payGhostRock(5),
```

Comment:
There is another cost type `payReduceableGRCost(amount)`, but I believe you used the correct one as it is not really a cost of the card. Or do you think this cost can be reduced? Is there actually a card that can reduce costs of action cards?

Reply:
I do not think there is another card that can lower the costs of an ability on a card rather than the cost of the card itself, though I could be wrong.

Reply:
Ok. I guess we will leave it like this. If there is a problem with it, we can fix later.